src/app/components/home/TransactionPanel.tsx

```diff
@@ -153,7 +153,10 @@ const getValuesAndSymbols = (tx: Transaction) => {
         try {
           const data = JSON.parse(tx.data[tx.data.length - 1].replace(/'/g, '"'));
           Object.keys(data).forEach(key => {
-            if (data[key] !== 0) {
+            if (isIUSDC(key)) {
```

Comment:
if (data[key] !== 0) {
              symbols.push(getContractName(key) || '');
              amounts.push(isIUSDC(key) ? convertIUSDC(new BigNumber(data[key])) : convertValue(data[key]));
            }

please change the code like this.

Reply:
Thanks for your feedback, much cleaner! Updated it